docs/swagger/spec.yml

```diff
@@ -1414,6 +1468,49 @@ definitions:
               - PARTIALFAILURE
               - QUEUED
               - PROCESSING
+  SceneGrid:
+    allOf:
+      - type: object
+        description: Geojson object
+        properties:
+          type:
+            type: string
+            description: Type of geojson feature. Expect 'FeatureCollection'
+          polygons:
```

Comment:
Should this be polygons? The design has numbers that represent the value in the grid -- could be better to return points for the center of the grid and values for the number of scenes in that grid